client/src/redux/app-mount-saga.js

```diff
@@ -2,23 +2,30 @@ import qs from 'query-string';
 import { put, takeEvery } from 'redux-saga/effects';
 import { createFlashMessage } from '../components/Flash/redux';
 
+function* clearSearchWithoutReRender() {
```

Comment:
```suggestion
function clearSearchWithoutReRender() {
```
Since there's no yield statement, this isn't really a generator function.